docs/index.md

```diff
@@ -651,21 +651,31 @@ The process begins with a QR code or `openid://` link. The only differences are:
     ```
 ---
 
-## Potential Extensions
+### Paper-based fallback: Health Card Links
+* Allows basic use of health cards for users without a smartphone
+* Allow smartphone-enabled users to print a usable backup
+* Allows full health card contents to be shared with a verifier
+* Requires that the verifier is online
```

Comment:
In other paper-based credential discussions I've heard approaches where the verifier could be offline with a cached list of trusted issuers and their public keys. The verifier could periodically connect to the network to refresh its cached list, but it wouldn't necessarily have to be online at the time of verification. There are trade-offs here when it comes to having an update to date list of trusted issuers.

I don't have a strong opinion on changing this bullet because it might be more complicated/confusing than it is worth.